pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -554,6 +555,105 @@ protected void internalSetNamespaceMessageTTL(int messageTTL) {
         }
     }
 
+    protected void internalSetAutoTopicCreationOverride(AsyncResponse asyncResponse, AutoTopicCreationOverride autoTopicCreationOverride) {
+        validateAdminAccessForTenant(namespaceName.getTenant());
+        validatePoliciesReadOnlyAccess();
+
+        if (!AutoTopicCreationOverride.isValidOverride(autoTopicCreationOverride)) {
+            throw new RestException(Status.PRECONDITION_FAILED, "Invalid configuration for autoTopicCreationOverride");
+        }
```

Comment:
We'd better complete the `asyncResponse` when there are RunTimeException occurs. Otherwise, the client just can get a `500` response but can't get any error messages.